src/en/news/blog/2025/rbd-live-migration/index.md

```diff
@@ -0,0 +1,478 @@
+_Scenario_: A organization must comply with local data residency regulations that
+require sensitive data to be stored within specific geographic boundaries.
+Data held in RBD images thus must be migrated from a general-purpose Ceph
+cluster to one dedicated to and within the regulated region.\
+_Benefit_: The live migration feature enables seamless relocation of RBD data
+without halting ongoing operations, ensuring compliance with regulations.
+
+* Multi-Cluster Load and Capacity Balancing\
+_Scenario_: An organization runs multiple Ceph clusters to handle high traffic
+workloads. To prevent overloading any single cluster, they redistribute
+RBD images among clusters as workload patterns shift.\
+_Benefit_: Live migration allows for efficient rebalancing of workloads across
+Ceph clusters, optimizing resource utilization and performance.
+
+* Dev/Test to Production Migration\
+_Scenario_: Developers run test environments on a dedicated Ceph cluster.
+After testing is complete, production-ready RBD images can be migrated to the
+production Ceph cluster without data duplication or downtime.\
+_Benefit_: Simplifies the process of promoting test data to production
+while maintaining data integrity.
+
+* Hardware Lifecycle Management\
+_Scenario_: A Ceph cluster is running on older hardware nearing the end of
+its lifecycle. The admin plans to migrate RBD images to a new Ceph cluster
+with upgraded hardware for better performance and reliability.\
+_Benefit_: Live migration facilitates a smooth transition from legacy to modern
+infrastructure without impacting application uptime.\
+_Note_: In many situations one can incrementally replace 100% of Ceph cluster
+hardware in-situ without downtime or migration, but in others it may be
+expedient or necessary to stand up a new,independent cluster and migrate
+data between the two.
+
+* Service Provider Multi-Tenancy\
+_Scenario_: A cloud service provider uses Ceph as backend storage and needs to
+migrate tenant data (RBD images) between clusters for reasons including tenant
+location preference, cost optimization, and internal service redistribution.\
+_Benefit_: Enables seamless data mobility while ensuring service
+continuity and meeting SLA requirements.
+
+* Cluster Version Upgrade\
+Scenario: A Ceph admin wants to upgrade their Ceph cluster to a newer major release but
+is concerned that the new release may introduce ugs or regressions.  Downgrading
+to an older Ceph release in-situ is generally not supported, so out of an abundance
+of caution the admin wishes to move RBD images to a separate cluster temporarily
+during the upgrade to ensure uninterrupted service.\
+_Benefit_: Live migration allows the customer to avoid downtime during the upgrade
+process and ensures seamless rollback if needed.
+
+* Global Data Replication\
+_Scenario_: An enterprise has Ceph clusters distributed across locations
+to improve latency for regional end users. They can migrate RBD images
```

Comment:
s/They can migrate RBD images/RBD images can be migrated/

This is a case in which the passive voice is preferable to the use of a bewildering pronoun.